tests/Unit/Evolution/Systems/NewtonianEuler/BoundaryCorrections/Test_Rusanov.cpp

```diff
@@ -76,19 +77,27 @@ struct DummyInitialData {
   };
 };
 
+namespace helpers = TestHelpers::evolution::dg;
+
 template <size_t Dim, typename EosType>
 void test(const size_t num_pts, const EosType& equation_of_state) {
   tuples::TaggedTuple<hydro::Tags::EquationOfState<EosType>> volume_data{
       equation_of_state};
-
-  TestHelpers::evolution::dg::test_boundary_correction_conservation<
+  tuples::TaggedTuple<
+      helpers::Tags::Range<NewtonianEuler::Tags::MassDensityCons>,
+      helpers::Tags::Range<
+          NewtonianEuler::Tags::SpecificInternalEnergy<DataVector>>>
+      ranges{std::array<double, 2>{{1e-6, 1.0}},
```

Comment:
I'm genuinely unsure of how small the lower bound here should be. Since ideally our evolution scheme should support down to something like `1e-30` should we lower to that?

Also `1e-6` -> `1.0e-6`

Reply:
I tried this out with the range (1e-30, 1e-29) to make sure I'd get something in the "very small" end of the range. Things seem ok. I'm squashing in a change of the lower bound to 1e-30. Squashed in the notation change too.